### DISCUSSION TITLE
Buffer overrun questions

### DISCUSSION THREAD

**Discussion by maardsma** [savonet/liquidsoap#1896] [Q&A]
I am getting buffer overruns on my input.http source. I don't know why. max is set to 10 seconds.

1) Is there a way to turn off buffer overrun logging in version 1.4.4?

2) Is there a method to flush/empty the buffer?

In my situation, I care more about low latency than about avoiding dropping audio. As everything starts up I seem to fill the buffer. I'd like my script to dump that buffer after everything is stable. Or skip the source ahead to the end of the current buffer.

**Comment by toots** [comment]
I believe my answer to https://github.com/savonet/liquidsoap/discussions/1897 probably has to do with this one as well.
